### PR TITLE
ADCM-1630 add tests for service host actions

### PR DESCRIPTION
**Pull request by skhomuti**

### CODE REVIEW

File: tests/functional/test_actions_on_host.py [hunk 1]
```diff
@@ -77,19 +85,93 @@ def test_availability_at_host_state(self, cluster: Cluster, provider: Provider):
...
+        action_in_object_is_absent(action_name, service)
+        run_host_action_and_assert_result(host_with_one_component, action_name)
+        run_host_action_and_assert_result(host_with_two_components, action_name)
+        run_host_action_and_assert_result(host_with_different_services, action_name)
+
+    def test_availability_at_state(self, cluster_with_service: Cluster, provider: Provider):
+        """
+        Test that service host action is available on specify service state
+        """
+        action_name = "action_on_host_state_installed"
```
Comment (vgorkavenko) [inline tests/functional/test_actions_on_host.py:128]: Const?

File: tests/functional/test_actions_on_host.py [hunk 2]
```diff
@@ -77,19 +85,93 @@ def test_availability_at_host_state(self, cluster: Cluster, provider: Provider):
...
+def action_in_object_is_present(action: str, obj: ObjTypes):
     with allure.step(f"Assert that action {action} is present in {_get_object_represent(obj)}"):
         try:
             obj.action(name=action)
         except ObjectNotFound as err:
             raise AssertionError(f"Action {action} not found in object {obj}") from err
 
 
-def action_in_object_is_absent(action: str, obj: Union[Cluster, Host]):
+def action_in_object_is_absent(action: str, obj: ObjTypes):
```
Comment (vgorkavenko) [inline tests/functional/test_actions_on_host.py:170]: Why not `Union[Cluster, Host, Service]`?

Comment (skhomuti) [reply tests/functional/test_actions_on_host.py:170]: it's a type alias.

File: tests/functional/test_actions_on_host.py (hunk 1 above)
Comment (skhomuti) [reply tests/functional/test_actions_on_host.py:128]: fix all consts

**Review by dgusakov — APPROVED** [review]

**Review by vgorkavenko — APPROVED** [review]

**Pull request merged**